light_curves/ML_AGNzoo.md

```diff
@@ -69,12 +71,10 @@ sys.path.append('code_src/')
 from AGNzoo_functions import (unify_lc, unify_lc_gp, stat_bands, autopct_format, combine_bands,
                       normalize_clipmax_objects, shuffle_datalabel, dtw_distance,
                       stretch_small_values_arctan, translate_bitwise_sum_to_labels, update_bitsums)
-from collections import Counter,OrderedDict
+from collections import Counter,OrderedDict,defaultdict
```

Comment:
Since python 3.7 dictionaries preserve order, so  I don't think you specifically need to use OrderDict anywhere.